doc/contributing/docs/api.rst

```diff
@@ -179,9 +179,11 @@ Configuration parameters
 
 Example:
 
-..  literalinclude:: ./_includes/confval_template.rst
+..  literalinclude:: /reference/configuration/cfg_basic.rst
     :language: rst
+    :lines: 219-231
 
 Result:
 
-..  include:: ./_includes/confval_template.rst
+..  literalinclude:: /reference/configuration/cfg_basic.rst
```

Comment:
The result looks incorrect:
![Uploading image.png…]()

Reply:
Thanks, fixed:
https://docs.d.tarantool.io/en/doc/fix-warnings/contributing/docs/api/#contributing-api-docs-confval-example